VDA-SBT/contracts/SoulboundNFT.sol

```diff
@@ -36,10 +36,10 @@ contract SoulboundNFT is VDAVerificationContract,
     EnumerableSet.StringSet private _sbtTypes;
 
     /**
-     * @notice Claimed SBT info by users
-     * @dev mapping of User => SBTType => UniqueId => tokenId
+     * @notice Claimed token id for SBTType & UniqueId
+     * @dev mapping SBTType => UniqueId => tokenId
      */
-    mapping(address => mapping(string => mapping(string => uint))) private _userInfo;
```

Comment:
Updated the internal variables. Before, `_userInfo` variable keeps the SBTType & uniqueID per users. In the `claimSBT()` function, it checked whether the combination SBTType & uniqueID was unique to the user. This means, any other user can claim the SBT with the same SBTType & uniqueID. So removed this variable and added `_sbtID` variable. Now the contract checks whether the combination of SBTType & uniqueID is uniques in the contract.